### PR TITLE
Render e-mails from string

### PR DESCRIPTION
**Pull request by rjkip**
https://www.pivotaltracker.com/story/show/83546778

### CODE REVIEW

**Comment by rjkip** [comment]
:facepunch: @DRvanR

File: src/Surfnet/StepupMiddleware/CommandHandlingBundle/Command/AbstractCommand.php [hunk 1]
```diff
@@ -25,7 +25,7 @@
     /**
      * @Assert\NotBlank()
      * @Assert\Type(type="string")
-     * @Assert\Regex(pattern="~^[0-9A-F]{8}-[0-9A-F]{4}-4[0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}$~")
+     * @Assert\Regex(pattern="~^[0-9a-f]{8}-[0-9a-f]{4}-4[0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$~")
```
Comment (DRvanR) [inline src/Surfnet/StepupMiddleware/CommandHandlingBundle/Command/AbstractCommand.php]: rather put an `i` at the end :wink:

**Comment by rjkip** [comment]
Done. @DRvanR 

df

**Comment by DRvanR** [comment]
oooh my

:+1:

**Pull request merged**